src/pmpo_c.h

```diff
@@ -8,49 +8,49 @@ extern "C" {
 typedef void* MPMesh_ptr;
 
 //initialize and finalize
-void polympo_initialize();
-void polympo_finalize();
+void polympo_initialize_f();
+void polympo_finalize_f();
 
 //create/delete MpMesh object
 MPMesh_ptr polympo_createMPMesh(int setMeshOption, int setMPOption);
```

Comment:
Should you also change this one `polympo_createMPMesh`?
 https://github.com/SCOREC/polyMPO/pull/20/files#diff-71b9f5e91f786b7ec9b39c381d0e64b321775edd56d721a9aaa65a2b5d2afd9fR15